src/ParserContext.php

```diff
@@ -13,6 +13,14 @@
 use type Facebook\Markdown\UnparsedBlocks\Context as BlockContext;
 use type Facebook\Markdown\Inlines\Context as InlineContext;
 
+//$parser_context->setSourceType(SourceType::TRUSTED|SourceType::SPONSORED|SourceType::USER_GENERATED_CONTENT)
```

Comment:
remove

Reply:
Sorry about all the commits! Meant to only push my recent one. Addressed everything